Lombiq.NodeJs.Extensions/scripts/lint-markdown.js

```diff
@@ -44,7 +44,7 @@ function getMarkdownPaths() {
     return findRecursively(
         rootDirectory,
         [/\.md$/i],
-        [/^node_modules$/, /^\.git$/, /^\.vs$/, /^\.vscode$/, /^\.idea$/, /^obj$/, /^bin$/, /^wwwroot$/]);
+        [/^node_modules$/, /^\.git$/, /^\.vs$/, /^\.vscode$/, /^\.idea$/, /^obj$/, /^bin$/, /^wwwroot$/, /^ThirdParty$/]); // codespell:ignore
```

Comment:
Document this here: https://github.com/Lombiq/NodeJs-Extensions/blob/dev/Lombiq.NodeJs.Extensions/Docs/Markdown.md Something like `Files in some special folders aren't linted, see the current configuration [here](link)`.